tests/components/airvisual/test_config_flow.py

```diff
@@ -37,6 +42,34 @@
         assert result["errors"] == {CONF_API_KEY: "invalid_api_key"}
 
 
+async def test_options_flow(hass):
+    """Test config flow options."""
+    conf = {CONF_API_KEY: "abcde12345"}
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="abcde12345",
+        data=conf,
+        options={CONF_SHOW_ON_MAP: True},
+    )
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.airvisual.async_setup_entry", return_value=True
+    ):
+        result = await hass.config_entries.options.async_init(config_entry.entry_id)
+
+        assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+        assert result["step_id"] == "init"
+
+        result = await hass.config_entries.options.async_configure(
+            result["flow_id"], user_input={CONF_SHOW_ON_MAP: False}
+        )
+
+        assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+        assert config_entry.options == {CONF_SHOW_ON_MAP: False}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5156575)